### ISSUE TITLE
The outputs of ProcessCitationList were empty

### ISSUE THREAD

**Issue by Kobayashi-Kei** [kermitt2/grobid#869]
I executed  ProcessCitationList By grobid_client_python.
The output files were generated but they were empty.

I just executed below command in README of  grobid_client_python Repository.
`grobid_client --input resources/test_txt/ --output resources/test_out/ processCitationList`

Log of grobid server is this.
```
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "GET /api/isalive HTTP/1.1" 200 4 "-" "python-requests/2.26.0" 39  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 1  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 3  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 1  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 2  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 2  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 5  
0:0:0:0:0:0:0:1 - - [05/12月/2021:03:03:32 +0000] "POST /api/processCitationList HTTP/1.1" 404 0 "-" "python-requests/2.26.0" 6
```  

grobid version  is 0.7.0

Is there any solution?

**Comment by kermitt2** [comment]
Hi @Kobayashi-Kei !

Thanks for using Grobid.

404 so the service does not exist on the server. Indeed, `processCitationList` has been introduced later and is available with 0.7.1-SNAPSHOT only currently. I plan to release 0.7.1 soon.  

So you would need to use the current master of Grobid to use this service.

**Comment by Kobayashi-Kei** [comment]
Hi @kermitt2 !

I cloned master of Grobid and I succeeded to execute `processCitationList`!

Grobid is great application!!
Thank you!
